randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -446,21 +467,29 @@ Extra - asset_id: 13
 > Door to Terminator Room; Heals? False
   * Layers: default
   * Normal Door to Terminator Room/Door to Green Pirates Shaft
-  > Door to Lower Mushrooms
-      Trivial
   > Door to Gold Four Hallway
-      Trivial
+      Any of the following:
+          Charge Beam or Ice Beam or Missile or Plasma Beam or Power Bombs or Spazer Beam or Super Missile or Wave Beam
+          Damage Boosting (Intermediate) and Normal Damage ≥ 61
+          Morph Ball Bombs and Morph Ball
+          Damage Boosting (Beginner) and Normal Damage ≥ 101
 
 > Door to Lower Mushrooms; Heals? False
   * Layers: default
   * Normal Door to Lower Mushrooms/Door to Green Pirates Shaft
-  > Door to Terminator Room
+  > Door to Gold Four Hallway
       Trivial
 
 > Door to Gold Four Hallway; Heals? False
   * Layers: default
   * Missile Door to Gold Four Hallway/Door to Green Pirates Shaft
   > Door to Terminator Room
+      Any of the following:
+          Charge Beam or Ice Beam or Missile or Plasma Beam or Power Bombs or Spazer Beam or Super Missile or Wave Beam
+          Damage Boosting (Intermediate) and Normal Damage ≥ 81
```

Comment:
Is 61 vs 81 intentional?

Reply:
Yes. Going down is much easier than going up. I wasn't able to go up with 3 hits only. 4 was my best. Hence the 61 down and 81 up.